packages/editor/templates/Coin/README.md

```diff
@@ -152,7 +152,7 @@ Coin.prototype.createTokens = function() {
         return;
     }
 
-    this.instance.mint(address, amount, { from: window.web3.eth.accounts[0], gas: 100000, gasPrice: 100000, gasLimit: 100000 },
+    this.instance.mint(address, amount, { from: window.web3.eth.accounts[0], gas: 100000, gasPrice: 100000, gasLimit: 100000 }, 
```

Comment:
Small potatoes but I'd remove the trailing space.

Reply:
```suggestion
    this.instance.mint(address, amount, { from: window.web3.eth.accounts[0], gas: 100000, gasPrice: 100000, gasLimit: 100000 },
```